.github/workflows/luacheck.yml

```diff
@@ -11,3 +11,11 @@ jobs:
         uses: actions/checkout@v3
       - name: Luacheck
         uses: lunarmodules/luacheck@v0
+
+      - name: Install EditorConfig Lint
+        uses: editorconfig-checker/action-editorconfig-checker@d4fca16fc71adef10fbe101903b654449fa9570c
+      - name: Check all files for linting issues (not fatally)
+        run: editorconfig-checker || true
+      - name: Check for linting issues in edited files only
+        run: git diff --cached --name-only --diff-filter=d origin/master | xargs --no-run-if-empty editorconfig-checker
```

Comment:
I could see this being useful, but my experience it it then means that lint issues show up in unrelated PRs because they touch a file and get show a bunch of errors. I think I'd rather just fix everything project wide or add the necessary ignores, then do a fill project check all the time.